demo.py

```diff
@@ -359,7 +359,7 @@ def download_file(url):
     filename = os.path.join(tempfile.gettempdir(),
                             hashlib.sha1(url.encode()).hexdigest() + ext)
     if os.path.exists(filename):  # avoid redownloading
-        logger.info("Skipping download of {}: file already exist in ".format(url, filename))
```

Comment:
nit: maybe only fix the bug in this PR?

Reply:
flake got bumped automatically, I think either we freeze it, or if we let it flexible we can embed those kind of small fixes in PRs

Reply:
requirements-dev.txt should be fully frozen, even for python libs: it's *not* used by the lib users, it's OK to freeze/pin.

Reply:
Ok it wasn't in requirement.dev so i moved it there and froze it